lld/COFF/MapFile.cpp

```diff
@@ -122,16 +122,9 @@ static void getSymbols(const COFFLinkerContext &ctx,
     if (!file->live)
       continue;
 
-    if (!file->thunkSym)
-      continue;
-
-    if (!file->thunkSym->isLive())
-      continue;
-
-    syms.push_back(thunkSym);
-
-    if (auto *impSym = dyn_cast_or_null<Defined>(file->impSym))
```

Comment:
So technically, it's not possible for `impSym` to be null, or to be anything else than a `Defined` here?